recipe/meta.yaml

```diff
@@ -20,31 +20,45 @@ requirements:
     - python
     - setuptools
     - setuptools_scm
+    - wheel
 
   run:
     - python
-    - numpy >=1.9
+    - numpy >=1.17
```

Comment:
I don't see where the bound of numpy is set, is that really required?

Reply:
I can't recall why I updated this.  I bet I just used glue core's requirement but that's not best practice.